src/components/_shared/RecordsTable/useSortBy.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import moment from 'moment';
+
+export const useSortByDate = array => {
+  const [sortBy, setSortBy] = useState('NEWEST');
```

Comment:
I would use an enum here

Reply:
@yonatanmk please can you provide a snippet/update the code - when you mentioned enums outside of TS I'm thinking;

`
const sortByOptions = {
 newest: 'NEWEST'
 ...
}
`

Reply:
I'm just not a fan of string comparisons. I would just go with something like
```
const sortEnum = Object.freeze({
  NEWEST: 'NEWEST',
  OLDEST: 'OLDEST'
})
```

and use that throughout the components